README.md

```diff
@@ -35,6 +35,26 @@ For a quick reminder of these rules, just say `@AutoDelete help`.
 
 If you need extra help, say `@AutoDelete adminhelp ... message ...` to send a message to the support guild.
 
+## DockerFile
+
+For self hosting convience, i have included a dockerfile to build off of.
```

Comment:
note to self: fix grammar after merge

Reply:
I changed the verbiage completely here, hopefully resolving most of the grammar mistakes